doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -737,6 +737,143 @@ Alternatively, you may specify the max capacity for OSDs within the Ceph
 configuration file under the respective [osd.N] section. See
 :ref:`ceph-conf-settings` for more details.
 
+Global Override of Max IOPS Capacity for multiple OSDs
+------------------------------------------------------
+
+Depending on the requirement, the max IOPS capacity of multiple OSDs may be
+overridden by a global config specification. This can be done in case multiple
```

Comment:
s/in case/when/

Reply:
Condensed the content to a single section. Please re-verify.